tests/backends/test_arviz.py

```diff
@@ -32,7 +32,11 @@
 from pymc.exceptions import ImputationWarning
 
 # Turn all warnings into errors for this module
```

Comment:
My suggestion was to also inline this.

Reply:
But now I'm getting to the nittiest of nits. :)

Reply:
But that's why the filterwarnings is there in the first place? Without the temporary filters it looks like
```python
# Turn all warnings ...
pytest.filterwarnings("error")
```

Reply:
Do I misunderstand how `filterwarnings` works? The arguments get stacked, right?

```python
pytestmark = pytest.mark.filterwarnings(
    # Turn all warnings into errors for this module
    "error",
    # except those related to https://github.com/arviz-devs/arviz/issues/2327
    "ignore:datetime.datetime.utcnow():DeprecationWarning",
)
```

Reply:
My point is, that without the temporary ignores, it makes more sense to have the comment above the call. That also explains why the call is there in the first place.

```python
# Turn all warnings ...
pytest.mark.filterwarnings("error")
```
Makes more sense than
# Turn all warnings ...
pytest.mark.filterwarnings(
  # Turn all warnings ...
  "error",
)
```

Reply:
I'm gonna take the nitpick hint and merge as is

Reply:
https://en.wikipedia.org/wiki/Law_of_triviality